src/services/pegout-status/pegout-status.service.ts

```diff
@@ -0,0 +1,41 @@
+import {getLogger, Logger} from "log4js";
+import {inject} from "@loopback/core";
+import {ServicesBindings} from "../../dependency-injection-bindings";
+import {PegoutStatus, PegoutStatusAppDataModel} from "../../models/rsk/pegout-status-data-model";
+import {PegoutStatusMongoDbDataService} from "../pegout-status-data-services/pegout-status-mongo.service";
+
+export class PegoutStatusService {
+    private logger: Logger;
+    private pegoutStatusDataService: PegoutStatusMongoDbDataService;
+
+    constructor(
+        @inject(ServicesBindings.PEGOUT_STATUS_DATA_SERVICE)
+            pegoutStatusDataService: PegoutStatusMongoDbDataService,
```

Comment:
We should use the interface everytime we can
```suggestion
            pegoutStatusDataService: PegoutStatusDataService,
```